server/src/main/java/org/spine3/server/entity/AbstractVersionableEntity.java

```diff
@@ -48,9 +49,9 @@
         extends AbstractEntity<I, S>
         implements VersionableEntity<I, S> {
 
-    private final Version version;
+    private Version version;
 
-    private EntityStatus status = EntityStatus.getDefaultInstance();
+    private Visibility visibility = Visibility.getDefaultInstance();
```

Comment:
Right. Because `init()` may not be called and we want to have a valid field in between.